internal/expr/parser.go

```diff
@@ -154,16 +159,37 @@ func (p *Parser) Parse(input string) (expr *ParsedExpr, err error) {
 	}()
 
 	p.init(input)
+	queryParts, err := p.parse(minParenLevel{set: false})
+	if err != nil {
+		return nil, err
+	}
+	return &ParsedExpr{queryParts}, nil
+}
+
+// minParenLevel represents the a parenthese level for the parse to stop
+// parsing at. If valid is set to false then there is no minimum level.
```

Comment:
Typo in `parenthese` and there is no member called `valid` anymore, the comment seems out of date.

Reply:
Changed.